src/js/manager.js

```diff
@@ -29,17 +34,26 @@ $(function () {
     }
   });
 });
+
 /*
  * function for change option for command selection
  */
-function changeOption() {
+function changeCommand() {
```

Comment:
nit: how about commandValueChangeHandler'? its name is not clear whether it's handler or not.